doc/sphinx/docs/python/tutorials/using_population.rst

```diff
@@ -111,8 +120,8 @@ some individual decision vector:
     >>> print(pop1.get_f()[0])
     [ 8.43469444]
 
-.. note:: Using the method :class:`~pygmo.core.population.set_xf` it is possible to avoid
-          triggering the fitness function evaluation, but it is also possible to inject
+.. note:: Using the method :class:`~pygmo.core.population.set_xf` or:class:`~pygmo.core.population.push_back` it is possible to avoid
```

Comment:
These are methods and not classes, and the ``()`` round brackets are missing at the end.